src/services/audit/auditService.ts

```
@@ -169,14 +169,28 @@ export async function logAudit(entry: AuditEntry): Promise<void> {
     timestamp: new Date().toISOString(),
   };
 
+  // For admin/break-glass audit entries ensure attribution is present.
+  // If validation fails we must NOT publish to RabbitMQ — save to outbox
+  // immediately so the test-suite and production do not allow malformed
+  // admin audit entries to be published.
+  try {
+    validateAdminAttribution(entry);
+  } catch (e) {
+    const reason = e instanceof Error ? e.message : String(e);
+    logger.warn("RabbitMQ audit publish failed: invalid admin attribution", { error: reason, eventType: entry.eventType });
```

ESLint

[error] Replace `·error:·reason,·eventType:·entry.eventType` with `⏎······error:·reason,⏎······eventType:·entry.eventType,⏎···`

(prettier/prettier)